flink-libraries/flink-table/src/test/scala/org/apache/flink/table/runtime/harness/HarnessTestBase.scala

```diff
@@ -553,6 +384,20 @@ class HarnessTestBase extends StreamingWithStateTestBase {
     stateField.get(generatedAggregation).asInstanceOf[DataView]
   }
 
+  def getGeneratedAggregationFields(
+      operator: AbstractUdfStreamOperator[_, _],
+      funcName: String,
+      funcClass: Class[_]): Array[Field] = {
+    val function = funcClass.getDeclaredField(funcName)
+    function.setAccessible(true)
+    val generatedAggregation =
+      function.get(operator.getUserFunction).asInstanceOf[GeneratedAggregations]
```

Comment:
Can we add a test case that check is the member variable of the test `GeneratedAggregations meeting the expectations? i.e: check the number of test variables or `NonWindowedAggregationHelper` which named `acc[Num]_distinctValueMap_dataview` ?